pkg/operator/api/api.go

```diff
@@ -37,7 +37,7 @@ type Server interface {
 
 type apiServer struct {
 	Client     client.Client
-	updateChan chan (*componentsapi.Component)
+	updateChan chan *componentsapi.Component
```

Comment:
lgtm

BTW, could it be detected by one linter in `golangci-lint`?

Reply:
Sorry， I don't find any linters match it in [golangci-lint.run](https://golangci-lint.run/usage/linters/)